### ISSUE TITLE
chain_name defaults to nil but needs to be defined to use chain

### ISSUE THREAD

**Issue by AlexeyDemidov** [zuazo/ssl_certificate-cookbook#24]
In following example (chain_name omitted) chain content is ignored and not included in chained site.com.pem.chained.pem. While chain_name default value is documented in README but it probably should specify that you need to explicitly define chain_name to use intermediate certificate.

```ruby
cert = ssl_certificate "site.com" do
    source 'data-bag'
    bag 'ssl_certs'
    key_item "site.com"
    key_item_key 'key'
    cert_item "site.com"
    cert_item_key 'cert'
    chain_item "site.com"
    chain_item_key 'chain'
  end
```

**Comment by zuazo** [comment]
Hi @AlexeyDemidov,

Thanks for the issue :wink:

I added a little explanation in the README. I hope it's enough.

**Comment by zuazo** [comment]
Documentation fix released in [`1.12.0`](https://github.com/zuazo/ssl_certificate-cookbook/releases/tag/1.12.0).
